From f0b6cf4a1b79bc6dc9860ded29d95a517b967a3d Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Sat, 19 Jul 2014 04:29:39 -0400
Subject: [pci] Fix pci_check_and_unmask_intx() comment typos

Message-id: <20140719042939.19438.85539.stgit@gir.stowe>
Patchwork-id: 86028
O-Subject: [RHEL7.1 PATCH 66/81] PCI: Fix pci_check_and_unmask_intx() comment typos
Bugzilla: 1110895
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110895
Upstream Status: ebd50b939c33c692a4914e66bf06a99b21ffa483

commit ebd50b939c33c692a4914e66bf06a99b21ffa483
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Tue Jan 14 17:10:39 2014 -0700

    PCI: Fix pci_check_and_unmask_intx() comment typos

    Fix typos in pci_check_and_unmask_intx() function comment.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/pci/pci.c b/drivers/pci/pci.c
index 2c72b70..268aaa1 100644
--- a/drivers/pci/pci.c
+++ b/drivers/pci/pci.c
@@ -2987,7 +2987,7 @@ bool pci_check_and_mask_intx(struct pci_dev *dev)
 EXPORT_SYMBOL_GPL(pci_check_and_mask_intx);
 
 /**
- * pci_check_and_mask_intx - unmask INTx of no interrupt is pending
+ * pci_check_and_unmask_intx - unmask INTx if no interrupt is pending
  * @dev: the PCI device to operate on
  *
  * Check if the device dev has its INTx line asserted, unmask it if not
-- 
1.7.1